lib/mix/tasks/docs.ex

```diff
@@ -101,4 +106,17 @@ defmodule Mix.Tasks.Docs do
       true -> docs
     end
   end
+
+  # At some point we need to mark :readme option as
```

Comment:
Same here. Just remove README, people can update their mix docs. Benefits of being before 1.0. :)